lib/devise/strategies/passwordless_authenticatable.rb

```diff
@@ -20,7 +20,9 @@ def authenticate!
             )
 
             UserMailer.sign_in_email(user, url).deliver_now
-            fail!("An email was sent to you with a magic link.")
+            fail!("Follow the instructions in your email to continue.")
+          else
+            raise ::ApplicationController::EmailNotFoundError
```

Comment:
I raise an exception here which I then catch in the controller so that I can render the new page that is in the designs